src/components/Banner/Banner.jsx

```diff
@@ -1,9 +1,8 @@
 // Node modules.
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
 // Relative imports.
-import AlertBox from '../AlertBox/AlertBox';
+import { VaAlert } from 'web-components/react-bindings';
```

Comment:
We're using the React binding so that we can use the `onCloseAlert` event handler prop.